integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwBigCMMiiDomain.java

```diff
@@ -0,0 +1,425 @@
+                .namespace(domNamespace))
+            .includeServerOutInPodLog(true)
+            .serverStartPolicy("IF_NEEDED")
+            .serverPod(new ServerPod()
+                .addEnvItem(new V1EnvVar()
+                    .name("JAVA_OPTIONS")
+                    .value("-Dweblogic.StdoutDebugEnabled=false"))
+                .addEnvItem(new V1EnvVar()
+                    .name("USER_MEM_ARGS")
+                    .value("-Djava.security.egd=file:/dev/./urandom ")))
+            .adminServer(new AdminServer()
+                .serverStartState("RUNNING")
+                .adminService(new AdminService()
+                    .addChannelsItem(new Channel()
+                        .channelName("default")
+                        .nodePort(0))))
+            .addClustersItem(new Cluster()
+                .clusterName("cluster-1")
+                .replicas(replicaCount)
+                .serverStartState("RUNNING"))
+            .configuration(new Configuration()
+                .opss(new Opss()
+                    .walletPasswordSecret(opssWalletPasswordSecretName))
+                .model(new Model()
+                    .domainType("JRF")
+                    .runtimeEncryptionSecret(encryptionSecretName))
+                .addSecretsItem(rcuAccessSecretName)
+                .introspectorJobActiveDeadlineSeconds(600L)));
+
+    return domain;
+  }
+
+  /**
+   * Verify Pod is ready and service exists for both admin server and managed servers.
+   * Verify EM console is accessible.
+   */
+  private void verifyDomainReady(String domainNamespace, String domainUid) {
+    String adminServerPodName = domainUid + "-admin-server";
+    String managedServerPrefix = domainUid + "-managed-server";
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace);
+
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Checking managed server service {0} is created in namespace {1}",
+          managedServerPrefix + i + "-c1", domainNamespace);
+      checkPodReadyAndServiceExists(managedServerPrefix + i + "-c1", domainUid, domainNamespace);
+    }
+
+    //check access to the em console: http://hostname:port/em
+    int nodePort = getServiceNodePort(
+        domainNamespace, getExternalServicePodName(adminServerPodName), "default");
+    assertTrue(nodePort != -1,
+        "Could not get the default external service node port");
+    logger.info("Found the default service nodePort {0}", nodePort);
+    String curlCmd1 = "curl -s -L --show-error --noproxy '*' "
+        + " http://" + K8S_NODEPORT_HOST + ":" + nodePort
+        + "/em --write-out %{http_code} -o /dev/null";
+    logger.info("Executing default nodeport curl command {0}", curlCmd1);
+    assertTrue(callWebAppAndWaitTillReady(curlCmd1, 5), "Calling web app failed");
+    logger.info("EM console is accessible thru default service");
+  }
+
+  private static Integer getDBNodePort(String namespace, String dbName) {
```

Comment:
Move this utility to utils/DbUtils.java,  we have the same code repeated in multiple ITClasses. 

$ grep "private static Integer getDBNodePort" *.java
ItCrossDomainTransaction.java:  private static Integer getDBNodePort(String namespace, String dbName) {
ItIstioCrossClusters.java:  private static Integer getDBNodePort(String namespace, String dbName) {
ItIstioCrossDomainTransaction.java:  private static Integer getDBNodePort(String namespace, String dbName)

Reply:
done